packages/massa-proto-cli/src/index.ts

```diff
@@ -39,23 +60,34 @@ function getDir(args: string[]): string {
   if (index >= 0 && index + 1 < args.length) {
     return args[index + 1];
   }
-  return '.';
+  return './helpers/';
 }
 
-function run(args: string[]) {
+async function run(args: string[]) {
   let files: IProtoFile[] = [];
   let mode = getGenMode(args);
   let address = getAddr(args);
   let out = getDir(args);
 
+  const client: Client = await ClientFactory.createCustomClient(
+  [
+      { url: publicApi, type: ProviderType.PUBLIC } as IProvider,
+      // This IP is false but we don't need private for this script so we don't want to ask one to the user
+      // but massa-web3 requires one
+      { url: publicApi, type: ProviderType.PRIVATE } as IProvider,
```

Comment:
I guess we should create an issue on massa-web3 about that...

Reply:
#233